src/common/utxobased/engine/types.ts

```diff
@@ -1,7 +1,20 @@
+import { asArray, asBoolean, asObject, asOptional, asString } from 'cleaners'
 import { EdgeSpendInfo } from 'edge-core-js/types'
 
 import { Input, Output } from '../keymanager/utxopicker/types'
 
+// Normalizes a URL to always be a websocket URL
+const asWebsocketUrl = (raw: unknown): string => {
+  const url = new URL(asString(raw))
+  return `wss://${url.host}/websocket`
+}
```

Comment:
We should de-dup this code in the `pluginState.js` file

Reply:
Instead: remove in favor of adding the `/websocket` where it's needed in the call-stack.